src/mantarray_desktop_app/ok_comm.py

```diff
@@ -70,10 +76,34 @@
     )
 
 
+def check_barcode_for_errors(barcode: str) -> str:
```

Comment:
This is a function used by the start_recording route and the return strings are error messages returned by the route if the barcode is invalid. I just moved it into ok_comm so I wouldn't need to rewrite the code. I think it would make sense to change it when we remove the barcode parameter from the start_recording route